draft-ietf-masque-h3-datagram.md

```diff
@@ -290,6 +290,10 @@ Data streams can be prioritized using any means suited to stream or request
 prioritization. For example, see {{Section 11 of
 ?PRIORITY=I-D.ietf-httpbis-priority}}.
 
+Data streams are subject to the flow control mechanisms of the underlying layers
+(for example, HTTP/2 stream flow control, HTTP/2 connection flow control, and
```

Comment:
```suggestion
(for example, HTTP/2 stream flow control, HTTP/3 connection flow control, and
```

Reply:
more accurately said as QUIC connection flow control. David added these examples, not sure how comprehensive we want to be.